package.json

```
@@ -33,7 +33,12 @@
         "storybook": "dev-pm start storybook",
         "test": "pnpm --recursive run test",
         "version": "$npm_execpath changeset version && pnpm install --lockfile-only",
-        "publish": "pnpm run build:packages && $npm_execpath changeset publish"
+        "publish": "pnpm run build:packages && $npm_execpath changeset publish",
+        "demo:browser": "run-p demo:browser:api demo:browser:admin demo:browser:site demo:browser:jaeger",
+        "demo:browser:api": "dotenv -- sh -c 'open-cli $API_URL/graphql'",
+        "demo:browser:admin": "dotenv -- sh -c 'open-cli $ADMIN_URL'",
+        "demo:browser:site": "dotenv -- sh -c 'open-cli $SITE_URL'",
+        "demo:browser:jaeger": "dotenv -- sh -c 'open-cli http://localhost:$JAEGER_UI_PORT'"
```

I'd suggest to switch browser and demo to be consistent with the `dev:*` scripts:

```suggestion
        "browser:demo": "run-p browser:demo:*",
        "browser:demo:api": "dotenv -- sh -c 'open-cli $API_URL/graphql'",
        "browser:demo:admin": "dotenv -- sh -c 'open-cli $ADMIN_URL'",
        "browser:demo:site": "dotenv -- sh -c 'open-cli $SITE_URL'",
        "browser:demo:jaeger": "dotenv -- sh -c 'open-cli http://localhost:$JAEGER_UI_PORT'"
```

We could also change the dev scripts to have demo first.